src/cargo/ops/cargo_new.rs

```diff
@@ -992,11 +994,25 @@ fn update_manifest_with_new_member(
             if was_sorted {
                 members.sort_by(|lhs, rhs| lhs.as_str().cmp(&rhs.as_str()));
             }
+            config.shell().status(
```

Comment:
Instead of two implementations (and putting it in this function),could we instead have this function return `bool` and put it in the caller?